src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/InfiniteScrollList.xaml.cs

```diff
@@ -82,13 +86,43 @@ internal InfiniteScrollList(Lazy<JoinableTaskFactory> joinableTaskFactory)
 
             BindingOperations.EnableCollectionSynchronization(Items, _list.ItemsLock);
 
-            ItemsView = new CollectionViewSource() { Source = Items }.View;
-            DataContext = ItemsView;
+            ItemsView = new CollectionViewSource() { Source = Items };
+            ListCollectionView itemsView = ItemsView.View as ListCollectionView;
+            itemsView.IsLiveFiltering = true;
+            itemsView.IsLiveGrouping = true;
+            itemsView.LiveFilteringProperties.Add($"{nameof(PackageItemViewModel.IsPackageVulnerable)}");
+            itemsView.LiveGroupingProperties.Add($"{nameof(PackageItemViewModel.PackageLevel)}");
```

Comment:
We can remove the string interpolation i.e. $""

```suggestion
            itemsView.LiveFilteringProperties.Add(nameof(PackageItemViewModel.IsPackageVulnerable));
            itemsView.LiveGroupingProperties.Add(nameof(PackageItemViewModel.PackageLevel));
```